### PR TITLE
virtio_fs:remove pjdfstest test cases on power

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID: 2066525
Signed-off-by: MiriamDeng <mdeng@redhat.com>

### CODE REVIEW

**Comment by MiriamDeng** [comment]
Current test cases
 (1/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: STARTED
- (1/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (2/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.auto: STARTED
\ (2/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.auto: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (3/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.auto.with_writeback: STARTED
| (3/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.auto.with_writeback: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (4/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.always: STARTED
/ (4/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.always: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (5/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.always.with_writeback: STARTED
- (5/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.always.with_writeback: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (6/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.none: STARTED
\ (6/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.none: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (7/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.none.with_writeback: STARTED
| (7/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.none.with_writeback: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (8/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_cache.auto: STARTED
/ (8/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_cache.auto: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (9/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_cache.auto.with_writeback: STARTED
- (9/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_cache.auto.with_writeback: CANCEL: Test cancelled due to --dry-run (0.0

**Comment by MiriamDeng** [comment]
@maxujun @PaulYuuu  Can you help to review the patch ? Thanks a lot

File: qemu/tests/cfg/virtio_fs_share_data.cfg [hunk 1]
```diff
@@ -182,6 +182,7 @@
                         fio_options += '--bs=4K --size=1G --iodepth=256 --numjobs=128 --runtime=1800 --thread'
                 - with_pjdfstest:
                     no s390 s390x
+                    no ppc64 ppc64le
```
Comment (PaulYuuu) [inline qemu/tests/cfg/virtio_fs_share_data.cfg:185]: Platforms limitation, suggest merging this into `no s390 s390x`.

style like:
```
no platform_1 platform_2
no os_1 os_2
no machine_type_1 machine_type2
```

**Comment by maxujun** [comment]
LGTM，Acked

**Comment by MiriamDeng** [comment]
Updated it, thanks

File: qemu/tests/cfg/virtio_fs_share_data.cfg (hunk 1 above)
Comment (MiriamDeng) [reply qemu/tests/cfg/virtio_fs_share_data.cfg]: Updated, could you help to review it ? Thanks a lot

**Comment by MiriamDeng** [comment]
Filtered jobs,
(1/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: STARTED
- (1/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (2/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.auto: STARTED
\ (2/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.auto: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (3/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.auto.with_writeback: STARTED
| (3/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.auto.with_writeback: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (4/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.always: STARTED
/ (4/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.always: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (5/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.always.with_writeback: STARTED
- (5/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.always.with_writeback: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (6/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.none: STARTED
\ (6/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.none: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (7/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.none.with_writeback: STARTED
| (7/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_fio.with_cache.none.with_writeback: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (8/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_cache.auto: STARTED
/ (8/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_cache.auto: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (9/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_cache.auto.with_writeback: STARTED
- (9/9) Host_RHEL.m8.u6.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_cache.auto.with_writeback: CANCEL: Test cancelled due to --dry-run (0.00

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**